src/components/UI/MainHeader.js

```diff
@@ -0,0 +1,13 @@
+import classes from "./MainHeader.module.css";
+import Navigation from "../Navigation/Navigation";
+
+const MainHeader = (props) => {
+  return (
+    <header className={classes["main-header"]}>
+      <h1>Workings hours</h1>
+      <Navigation onLogout={props.onLogout} />
```

Comment:
no need to give navigation this props since it is not used. the navigation uses context